### PR TITLE
Added ApplicationInsights to public site to enable usage tracking

### PR DESCRIPTION
**Pull request by joschemd**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Added Application Insights telemetry to public fabric site

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/12114)

### CODE REVIEW

File: apps/fabric-website/index.html [hunk 1]
```diff
@@ -53,6 +53,80 @@
         outline: none;
       }
     </style>
+    <script type="text/javascript">
```
Comment (ecraig12345) [inline apps/fabric-website/index.html]: Adding this here is okay, but there are a couple other locations you'll need to add it to actually make it be used on the public site (this file is only used for the PR deploy of the website and a couple other secondary locations). 

The file that's actually used on the public site is in a private repo which I can point you to, but fabric-website/homepage.htm is a mirror of that file. So typically we check in updates to homepage.htm (and index.html) here first, then port them to the private repo.

**Comment by ecraig12345** [comment]
@aneeshack4 We can ignore the screener failure in this case since there's no way it could be caused by this change. Once homepage.htm is also updated I'll merge it.

File: apps/fabric-website/index.html (hunk 1 above)
Comment (joschemd) [reply apps/fabric-website/index.html]: Completed .. will now add to the public-internal

**Comment by micahgodbolt** [comment]
@joschemd is working with dev.msft to get the rest of this hooked up

**Pull request merged**